### ISSUE TITLE
Increase unit test coverage to over 80%

### ISSUE THREAD

**Issue by millerg23** [atorus-research/xportr#82]
@bms63 we recently went through autovalidate process for `admiralroche`, and `xportr` was flagged as not having enough test coverage to reach our minimum threshold of 80%. I had a quick look and it looks like `xportr` needs 39 more lines covered by unit tests.
We have temporarily removed `xportr` from admiralroche package, but would be keen to add back in for our next release in June.
Appreciate everyone is busy, however I have a Roche colleague who is keen to bridge the test coverge gap! Please let me know if this would help.

**Comment by adchan11** [comment]
Hi @bms63 , I know you previously mentioned you were gathering a working group for xportr to bring it up to production level so in addition to that issue that I previously created regarding documentation of xportr functions, I can also assist in this issue! Feel free to reach out, thanks.

**Comment by bms63** [comment]
@adchan11 -thanks!

@elimillera Let's get the band back together and do this package proper!

**Comment by elimillera** [comment]
@bms63 Absolutely. pharmaverse is also doing a review of the package so we should be able to knock everything out at once. 

I'm back in the office on the 12th so we can start at that point.

**Comment by bms63** [comment]
![image](https://user-images.githubusercontent.com/10111024/232837022-48524ab1-87aa-434b-a7ef-bf4c11636e1c.png)

**Comment by bms63** [comment]
Please let us know what section of code you tackle for tests so others don't work on the same thing.

**Comment by averissimo** [comment]
With the current PRs the code coverage will nudge just past 80% 🎉

![image](https://user-images.githubusercontent.com/211358/233215619-404b7785-1c9d-4e0d-89f5-9964598121ac.png)

**Comment by bms63** [comment]
Wow!  Ya'll are amazing!  Can we keep going to ~100%?!?  We don't have a tight deadline so if we need to re-write some functions to increase testing opportunities, then lets do it!
